### PR TITLE
Removing old Analytics content from Setup and Reference sections

### PR DESCRIPTION
**Pull request by nilminiwso2**
- Installed and Setup section
- Reference section

### CODE REVIEW

**Review by samgnaniah — APPROVED** [review]

File: en/docs/install-and-setup/setup/reference/common-runtime-and-configuration-artifacts.md [hunk 1]
```diff
@@ -1,35 +1,23 @@
...
+These are directories in API Manager that includes deployable files, which are valid from a specified date and time at runtime.
 
-2. `<API-M_HOME>/repository/tenants` - This is only used when the deployment involves multi-tenancy. For more information, see [Configuring Multiple Tenants](https://docs.wso2.com/display/AM210/Configuring+Multiple+Tenants)
+-   `<API-M_HOME>/repository/deployment/server` -  Contains webapps that are related to customizing WSO2 API Manager during deployment. Required for deploying a super tenant. Also contains Synapse configurations and custom sequences, throttling execution plans.
 
-3. `<API-M_HOME>/solr` - Contains files for Apache solr indexing. For additional information, see [Add Apache Solr-Based Indexing](https://docs.wso2.com/display/AM210/Add+Apache+Solr-Based+Indexing)
+-   `<API-M_HOME>/repository/tenants` - This is only used when the deployment involves multi-tenancy. For more information, see [Configuring Multiple Tenants](https://docs.wso2.com/display/AM210/Configuring+Multiple+Tenants)
```
Comment (tishan89) [inline en/docs/install-and-setup/setup/reference/common-runtime-and-configuration-artifacts.md:16]: this link is from 2.1.0 documentation

File: en/docs/reference/customize-product/extending-api-manager/extending-gateway/writing-custom-handlers.md [hunk 2]
```diff
@@ -37,10 +37,6 @@ Let's see what each handler does:
  the token is of type `Production` or `Sandbox` and sets `MessageContext` variables as appropriate.
  - **APIThrottleHandler:** Throttles requests based on the throttling policy specified by the `policyKey` property. 
  Throttling is applied both at the application level as well as subscription level.
- - **APIMgtUsageHandler:** Publishes events to WSO2 Stream Processor (WSO2 SP) for collection and analysis of statistics.
```
Comment (tishan89) [inline en/docs/reference/customize-product/extending-api-manager/extending-gateway/writing-custom-handlers.md:40]: We have a new handler in APIM analytics. I think we have to add that here replacing this one

File: en/docs/reference/customize-product/extending-api-manager/extending-gateway/writing-custom-handlers.md [hunk 3]
```diff
@@ -94,11 +90,6 @@ before `</Handlers>`.
 
 2.  Restart API Manager.
 
-!!! note
-    To perform analytics with the logs, see 
-    [Analyzing the Log Overview]({{base_path}}/observe/api-manager-analytics/analyzing-the-log-overview) .
```
Comment (tishan89) [inline en/docs/reference/customize-product/extending-api-manager/extending-gateway/writing-custom-handlers.md:99]: Let's point this to monitoring logs section under observe

File: en/docs/reference/guides/message-flow-in-the-api-manager-gateway.md [hunk 4]
```diff
@@ -13,23 +13,19 @@ The Gateway of an API Manager deployment is responsible for the main business fu
 
 The handlers are request and response interceptors. The list of API handlers in WSO2 API-M are as follows:
 
--   APIMgtLatencyStatsHandler
 -   CORSRequestHandler
 -   APIAuthenticationHandler
 -   ThrottleHandler
--   APIMgtUsageHandler
```
Comment (tishan89) [inline en/docs/reference/guides/message-flow-in-the-api-manager-gateway.md:20]: Let's add the new handler name here

File: en/docs/reference/guides/message-flow-in-the-api-manager-gateway.md [hunk 5]
```diff
@@ -13,23 +13,19 @@ The Gateway of an API Manager deployment is responsible for the main business fu
...
 -   APIMgtGoogleAnalyticsTrackingHandler
 -   APIManagerExtensionHandler
 
 Each handler performs a specific task as mentioned in the table below. Note that some handlers are functional both at the inflow and outflow of messages.
 
 | Handler                              | Inflow                                                              | Outflow                                                         |
 |--------------------------------------|---------------------------------------------------------------------|-----------------------------------------------------------------|
-| APIMgtLatencyStatsHandler            | Publish request and response latencies, if analytics is enabled     | Publish request and response latencies, if analytics is enabled |
```
Comment (tishan89) [inline en/docs/reference/guides/message-flow-in-the-api-manager-gateway.md:28]: Let's add the new handler name and details here

File: en/docs/reference/samples/api-rate-monetization-sample.md [hunk 6]
```diff
@@ -35,6 +35,4 @@ To monetize APIs using your subscription models, do the following
 -   Connect to a billing engine
 -   Connect the API manager node to the billing engine to charge according to the API usage and bandwidth consumption.
 
-For more details, see [Enabling Monetization of APIs-]({{base_path}}/design/api-monetization/monetizing-an-api).
-
-To view API usage and statistics, configure WSO2 API-M Analytics. For more details, see [Configuring APIM Analytics]({{base_path}}/observe/api-manager-analytics/configuring-apim-analytics/#configuring-apim-analytics).
```
Comment (tishan89) [inline en/docs/reference/samples/api-rate-monetization-sample.md:40]: This line is needed. Lets point the users into configure section under observe

File: en/docs/reference/customize-product/extending-api-manager/extending-gateway/writing-custom-handlers.md (hunk 2 above)
Comment (ruks) [reply en/docs/reference/customize-product/extending-api-manager/extending-gateway/writing-custom-handlers.md:40]: New handler can be found here https://github.com/wso2/carbon-apimgt/blob/master/components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/analytics/AnalyticsMetricsHandler.java

Comment (nilminiwso2) [reply en/docs/reference/customize-product/extending-api-manager/extending-gateway/writing-custom-handlers.md:40]: @ruks - Can you update this PR or type in the description for this handler?

File: en/docs/reference/customize-product/extending-api-manager/extending-gateway/writing-custom-handlers.md (hunk 3 above)
Comment (nilminiwso2) [reply en/docs/reference/customize-product/extending-api-manager/extending-gateway/writing-custom-handlers.md:99]: done.

File: en/docs/reference/customize-product/extending-api-manager/extending-gateway/writing-custom-handlers.md (hunk 2 above)
Comment (nilminiwso2) [reply en/docs/reference/customize-product/extending-api-manager/extending-gateway/writing-custom-handlers.md:40]: Will send a new PR for this.
Will merge this PR now.

File: en/docs/reference/guides/message-flow-in-the-api-manager-gateway.md (hunk 5 above)
Comment (nilminiwso2) [reply en/docs/reference/guides/message-flow-in-the-api-manager-gateway.md:28]: will send a new PR with the new handler info.

File: en/docs/reference/guides/message-flow-in-the-api-manager-gateway.md (hunk 4 above)
Comment (nilminiwso2) [reply en/docs/reference/guides/message-flow-in-the-api-manager-gateway.md:20]: Will send a new pr with the new handler info.

**Pull request merged**